website/routes.py

```diff
@@ -220,17 +221,29 @@ def ember_app(path=None):
     fp = path or 'index.html'
 
     ember_app = None
+    strip_prefix = True
 
     for k in EXTERNAL_EMBER_APPS.keys():
         if request.path.strip('/').startswith(k):
             ember_app = EXTERNAL_EMBER_APPS[k]
             break
 
+    for asset_prefix in EMBER_ASSET_ROUTE_PREFIXES:
+        if request.path.strip('/').startswith(asset_prefix) and EXTERNAL_EMBER_APPS.get('ember_osf_web'):
+            ember_app = EXTERNAL_EMBER_APPS['ember_osf_web']
+            strip_prefix = False
+            fp = asset_prefix + '/' + fp
```

Comment:
Minor: Per COSdev, prefer str.format to concatenation. str.join would be fine here, too.

```python
fp = '/'.join(asset_prefix, fp)
```

https://cosdev.readthedocs.io/en/latest/style_guides/python.html#string-formatting

Certainly not a blocker though.

Reply:
Lol, our eslint rules for Ember apps actually don't even allow string concatenation, requiring the use of template literals (or a join as you suggested) instead. It's funny how our brains can totally switch modes when working in a different language/framework/part of the stack. I guess I should read the back-end style guide...